src/elements/content-sidebar/SidebarUtils.js

```diff
@@ -29,8 +29,8 @@ class SidebarUtils {
      * @return {Boolean} true if we should render
      */
     static canHaveDetailsSidebar({ detailsSidebarProps = {} }: ContentSidebarProps): boolean {
-        const { hasProperties, hasAccessStats, hasClassification, hasVersions, hasNotices } = detailsSidebarProps;
-        return !!hasProperties || !!hasAccessStats || !!hasClassification || !!hasVersions || !!hasNotices;
+        const { hasProperties, hasAccessStats, hasClassification, hasNotices } = detailsSidebarProps;
```

Comment:
Should this check remain if we're still passing `hasVersions` into the DetailsSidebar component?